### PR TITLE
update smart-socket to 1.5.6-SNAPSHOT

### PR DESCRIPTION
**Pull request by smthing**

### CODE REVIEW

File: frameworks/Java/smart-socket/src/main/java/org/smartboot/http/FortunesHandler.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        ve.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
+        ve.setProperty("eventhandler.escape.html.match", "/^(?!\\$\\!?unesc_).*/");
+        ve.init();
+        // 获取模板文件
+        Template template = ve.getTemplate("fortunes.vm", "MS932");
+        VelocityContext ctx = new VelocityContext();
+        List<Fortune> fortunes = new ArrayList<>();
+        fortunes.add(new Fortune(1, "<ab>"));
+        ctx.put("fortunes", fortunes);
+        ctx.put("text", "フレームワークのベンチマーク");
```
Comment (nbrady-techempower) [inline frameworks/Java/smart-socket/src/main/java/org/smartboot/http/FortunesHandler.java:63]: Why is this here?

Comment (smthing) [reply frameworks/Java/smart-socket/src/main/java/org/smartboot/http/FortunesHandler.java:63]: The file is still under local debugging and is not yet effective

Comment (nbrady-techempower) [reply frameworks/Java/smart-socket/src/main/java/org/smartboot/http/FortunesHandler.java:63]: Thanks @smthing. Can you please remove this file if it's not being used for this test.

Comment (smthing) [reply frameworks/Java/smart-socket/src/main/java/org/smartboot/http/FortunesHandler.java]: 👌
